To-do list/Controllers/Cell/TaskTableViewCell.swift

```diff
@@ -15,21 +15,27 @@ class TaskTableViewCell: UITableViewCell {
     
     weak var delegate: TaskTableViewCellDelegate?
     
-    @IBOutlet weak var titleLabel: UILabel!
-    @IBOutlet weak var descriptionLabel: UILabel!
-    @IBOutlet weak var deadlineLabel: UILabel!
-    @IBOutlet weak var statusButtonOutlet: UIButton!
-    
- 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-    }
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var deadlineLabel: UILabel!
+    @IBOutlet private weak var statusButtonOutlet: UIButton!
```

Comment:
you should not use "Outlet" for any UI element. Raname to "statusButton"

Reply:
done